prime-router/src/main/kotlin/azure/Event.kt

```diff
@@ -20,11 +20,11 @@ abstract class Event(val eventAction: EventAction, val at: OffsetDateTime?) {
         TRANSLATE, // Deprecated
         BATCH,
         SEND,
-        WIPE,
+        WIPE, // Deprecated
         NONE,
```

Comment:
?  Are we no longer planning to Wipe data?

Reply:
Right. Going to be done by delete policy.